system/jlib/jthread.cpp

```diff
@@ -1322,7 +1322,7 @@ static void CheckAllowedProgram(const char *prog,const char *allowed)
         if (WildMatch(head.str(),list.item(i)))
             return;
     }
-    ERRLOG("Unauthorized pipe program(%s)",head.str());
+    OERRLOG("Unauthorized pipe program(%s)",head.str());
```

Comment:
Possibly AERRLOG should this  be audited?  Jake what are the rules for audit messages?